tests/run/class_redefine.py

```diff
@@ -1,11 +1,10 @@
-
-class set(object):
+class set:
     def __init__(self, x):
         self.x = x
 
-SET = set([1])
+SET = {1}
```

Comment:
This is changing what's being tested. Note that the name `set` has been redefined a few lines above